### ISSUE TITLE
targets for bounty missions either don't spawn, or all spawn at once

### ISSUE THREAD

**Issue by flowers9** [endless-sky/endless-sky#9231]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

At some systems I'll pick up a number of bounty missions (3-4).  When I jump to a nearby system, either none of the targets will be present, or all of them will be.  It's like there's one random number that's being used for all of them.

I'm seeing this a lot in Remnant space, with 4-5 Korath of various makes showing up at once, as the bounties from each of the three planets all spawn at once.

### Steps to Reproduce

1. Get a high combat rating.
2. Go to planets until you get one with a lot of bounty missions.
3. Accept all the missions.
4. Jump to nearby systems until everything shows up at once.

### Expected Behavior

I'd expect a random number of targets to be present, based on their general chance to show up.  That is, each bounty should independently show up (or not).

### Operating System

Ubuntu 22.04.3 LTS

### Game Source

Built from source

### Game Version

e37732d (well, close to it)

### Additional Information

I tried hunting down the code that handles this, but haven't found it yet.

**Comment by xX-Dillinger-Xx** [comment]
I can confirm that this behaviour does happen, but I don't think it is a bug.
Some bounty missions indicate that the 'target' is within one jump away and some are within two jumps. 
The more bounty missions you except at the same time, the higher your odds they will overlap each other. There are only so many systems to choose _within one or two jumps_

**Comment by xX-Dillinger-Xx** [comment]
Should there be a warning that pops up if you try and accept a second bounty mission? Something along the lines of;
`JOB BOARD ASSISTANT: 
You have accepted more then one bounty mission, I sure hope you have what it takes to take on that many ships.
If you choose to proceed I wish you luck.`

**Comment by Amazinite** [comment]
I don't think it makes sense to display such a warning. It seems like it would be more of an annoyance than helpful to me.

Seeing as how this is basically expected behavior when there are so few systems to choose from, I'm closing this issue.
